README.md

```diff
@@ -103,9 +103,14 @@ To install stackrox-central-services, you will need a secure password. This pass
 ```sh
 ROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
 ```
+
 From here, you can install stackrox-central-services to get Central and Scanner components deployed on your cluster. Note that you need only one deployed instance of stackrox-central-services even if you plan to secure multiple clusters.
 
-If you're installing on a reasonably sized cluster, use the default installation command:
+To perform the installation, choose one of the following commands depending on your cluster size.
+
+#### Default Central Installation
+
+If you're installing in a reasonably sized cluster, use the default installation command:
```

Comment:
The change here is the 4th word: "on" vs "in".  I realized the headers I had removed used "installation _in_", so I changed this new text to match that.